test/codec.ts

```diff
@@ -479,14 +477,12 @@ describe('codec', () => {
     it('should encode structs', () => {
       const value = codec.Struct.fromJSON({a: 'b', c: 'd'});
       const encoded = codec.encode(value);
-      assert.deepStrictEqual(encoded, ['b', 'd']);
+      assert.deepStrictEqual([].concat(encoded.slice()), ['b', 'd']);
```

Comment:
@ofrobots I am not wild about this test change, but I can't see how my change above would require it.  Can you spot something with my es6-ification of the struct?

Reply:
Which change above are you referring to? How was the compiler complaining with the original form? Sorry, I am not familiar with the code here.